code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -89,23 +89,29 @@
 	new src.foldable(get_turf(src))
 	qdel(src)
 
-/obj/item/weapon/storage/box/survival/
-	New()
-		..()
-		new /obj/item/clothing/mask/breath( src )
-		new /obj/item/weapon/tank/emergency_oxygen( src )
+/obj/item/weapon/storage/box/survival
+	autodrobe_no_remove = 1
 
-/obj/item/weapon/storage/box/vox/
-	New()
-		..()
-		new /obj/item/clothing/mask/breath( src )
-		new /obj/item/weapon/tank/emergency_nitrogen( src )
+/obj/item/weapon/storage/box/survival/New()
+	..()
+	new /obj/item/clothing/mask/breath( src )
+	new /obj/item/weapon/tank/emergency_oxygen(src)
+	for(var/thing in contents)
+		autodrobe_no_remove = 1

```

Comment:
Were you trying to set this var on the object? This just sets it on the box a bunch of times.